omnigibson/transition_rules.py

```diff
@@ -261,13 +261,8 @@ def condition(self, individual_objects, group_objects):
             return False
 
         contact_list = slicer_obj.states[ContactBodies].get_value()
-        sliced_link_paths = {link.prim_path for link in sliced_obj.links.values()}
-        hit_obj = False
-        for c in contact_list:
-            if not set(c).isdisjoint(sliced_link_paths):
-                hit_obj = True
-                break
-        if not hit_obj:
+        sliced_links = set(sliced_obj.links.values())
+        if contact_list.isdisjoint(sliced_links):
```

Comment:
I thought contact_list is a list of tuples, where each tuple is a pair of contact link prim paths?

Reply:
oh, good point! I forgot i refactored contact_bodies to make it more efficient -- it also returns now the (set of) actual rigid bodies it's in contact with, rather than the tuple of string values.

Reply:
pushed changes